python/py-wikipedia-api/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        martin-majlis Wikipedia-API 0.5.4
+name                py-wikipedia-api
+revision            0
+supported_archs     noarch
+license             MIT
+
+python.versions     37 38 39 310
```

Comment:
do you need the older Python versions? If not, please only add the `310` at this point

Reply:
Hi @reneeotten,

I have removed the older versions of python.

Thanks,

-ranga